### DISCUSSION TITLE
Jobs list is not maintained properly

### DISCUSSION THREAD

**Discussion by ravin-accelq** [go-co-op/gocron#524] [General]
Hi,
``` job, err := Scheduler.Every(1).Millisecond().LimitRunsTo(1).Tag(tag).Do(task) ```

I got this code to trigger my child jobs only once. Then I use below function to track my active jobs. Seems like the Jobs list is empty since the LimitToRun is set to 1.

Can I get a function where I can track all the active jobs via tags, even though they are scheduled to run only once.

Even ```FindJobsByTag``` behaves the same way, immediately removing it from list.

```
func getActiveJobsByTag(tag string) *gocron.Job {
	jobs := Scheduler.Jobs()
	for _, job := range jobs {
		if job.Tags()[0] == tag {
			return job
		}
	}
	return nil
}
```
Ideally should it not track the jobs under Scheduler and jobs under Executor?

**Comment by JohnRoesler** [comment]
From the docs:

```
// LimitRunsTo limits the number of executions of this job to n.
// Upon reaching the limit, the job is removed from the scheduler.
//
// Note: If a job is added to a running scheduler and this method is then used
// you may see the job run more than the set limit as job is scheduled immediately
// by default upon being added to the scheduler. It is recommended to use the
// LimitRunsTo() func on the scheduler chain when scheduling the job.
// For example: scheduler.LimitRunsTo(1).Do()
func (j *Job) LimitRunsTo(n int) {
```

The key comment:
> Upon reaching the limit, the job is removed from the scheduler.
